install_sct

```diff
@@ -29,13 +29,12 @@
 # Authors: PO Quirion, J Cohen-Adad, J Carretero
 # License: see the file LICENSE.TXT
 
-# Change directory to where the script is, for the duration of the script.
-# This lives outside the subshell below to guarantee the corresponding
-# popd gets run, no matter how the installation script exits.
-pushd -- "${BASH_SOURCE[0]%/*}/" >/dev/null || exit 1
 
 ( # | tee install_sct_log.txt
 
+# Change directory to where the script is, for the duration of the script.
+cd "${BASH_SOURCE[0]%/*}/" || exit 1
```

Comment:
If memory serves, it was based on [a BashFAQ article](http://mywiki.wooledge.org/BashFAQ/028) which strongly argues against using `$0`. I don't think we're in a situation where this matters overmuch, honestly, but it seemed better to follow the recommendation.

I usually prefer to avoid bash-isms, but when the she-bang line at the top of the script specifies bash I don't try too hard.

Reply:
Further reading: https://stackoverflow.com/a/35006505

```bash
#!/bin/bash
echo "[$0] vs. [${BASH_SOURCE[0]}]"
```

```console
$ bash ./foo
[./foo] vs. [./foo]

$ ./foo
[./foo] vs. [./foo]

$ . ./foo
[bash] vs. [./foo]
```

Possibly also of interest: https://stackoverflow.com/a/54755784

> If you want to make your script both bash and zsh-compatible you can use `${BASH_SOURCE[0]:-${(%):-%x}}`. The resulting value will be taken from `BASH_SOURCE[0]` when it's defined, and `${(%):-%x}}` when `BASH_SOURCE[0]` is not defined.

Reply:
Like @mguaypaq says, though:

> I don't think we're in a situation where this matters overmuch, honestly, but it seemed better to follow the recommendation.

Especially given that we recommend in our docs to run `install_sct` using:

```
./install_sct
```

I would be very surprised if we encountered a user who wanted to run `. ./install_sct` or some other nonstandard way of invoking the install script. So, it would make sense to me to go back to `$0`?